src/Framework/App.Runtime/src/Microsoft.AspNetCore.App.Runtime.csproj

```diff
@@ -427,10 +438,6 @@ This package is an internal implementation of the .NET Core SDK and is not meant
         File="$(CrossgenToolDir)PlatformAssembliesPathsCrossgen2.rsp"
         Overwrite="true" />
 
-    <ItemGroup>
-      <RuntimePackageFiles Include="$(RuntimePackageRoot)runtimes\**\*" />
```

Comment:
Either move the new code down here or change this target to depend on `FilterUnwantedContent` instead of `_ExpandRuntimePackageRoot`.

Reply:
Did the latter